index.d.ts

```diff
@@ -0,0 +1,82 @@
+/// <reference types="node"/>
+import {Readable as ReadableStream} from 'stream';
+
+/**
+Checks if a `Buffer` contains a JPEG image that is [progressive](http://www.faqs.org/faqs/jpeg-faq/part1/section-11.html).
+
+@param buffer - Buffer of a JPEG image. Must be at least `65535` bytes when the file is larger than that.
+@returns Whether the `buffer` is a progressive JPEG image.
+
+@example
+```
+import {promisify} from 'util';
+import {readFile} from 'fs';
+import isProgressive = require('is-progressive');
+
+const readFileP = promisify(readFile);
+
+(async () => {
+	const buffer = await readFileP('baseline.jpg');
+	console.log(await isProgressive.buffer(buffer));
+	//=> false
+})();
+```
+*/
+export function buffer(buffer: Buffer): boolean;
```

Comment:
Oops. I missed this. These should not be named exports. They should be in a `declare const`.